fastlane_bot/helpers/poolandtokens.py

```diff
@@ -476,6 +485,67 @@ def decimal_converter(idx):
                 "descr": self.descr,
                 "params": self._params,
             }
+
+            strategy_typed_args += [typed_args]
+
+        #### MODIFICATION LOGIC HERE >>>>>
+        # Only overlapping strategies are selected for modification
+        if len(strategy_typed_args) == 2:
+            
+            is_overlapping = False
+            pmarg_threshold = Decimal("0.01") # 1%  # WARNING using this condition alone can included stable/stable pairs incidently
+
+            # evaluate that the marginal prices are within the pmarg_threshold
+            pmarg0, pmarg1 = [x['p_marg'] for x in strategy_typed_args]
+            pmarg0_inv = 1/pmarg0  # one of the orders must be flipped since prices are always dy/dx - but must flip same geomean_pmarg later
+            percent_component = pmarg_threshold * max(pmarg0_inv, pmarg1)
```

Comment:
It feels like instead of using `max(pmarg0_inv, pmarg1)`, you should use either one of these two values, based on the direction in which you are measuring the relative difference, which could lead to different results.

For example:
- `percent_component_met = abs(100 - 99) <= 0.01 * max(100, 99)` --> `True`
- `percent_component_met = abs(100 - 99) <= 0.01 * min(100, 99)` --> `False`

Of course, this means that the current method needs to be aware of the arb in context (i.e., it cannot be oblivious to the direction of the arbitrage being inspected).

I'm not sure if this makes sense, so FYI.

Reply:
Good suggestion but we wont implement for two reasons:
1) At this stage in the process we have no way of knowing the direction of the arb
2) This threshold is somewhat arbitrary and approximate solutions are ok. We expect the issue to occur around the 1% mark not necessarily a hard boundary and further investigation is ongoing